vm_images/README

```diff
@@ -1,2 +0,0 @@
-VM disk images belong in this folder.
```

Comment:
This will remove the directory `vm_images` from the git repo.
Edit the main README and the inventories examples to reflect this change or add a .gitkeep

Reply:
I edited the main README to indicate the user that he needs to create the folder first